pkg/api/addons.go

```diff
@@ -24,18 +24,18 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 	cloudSpecConfig := cs.GetCloudSpecConfig()
 	k8sComponents := K8sComponentsByVersionMap[o.OrchestratorVersion]
 	specConfig := cloudSpecConfig.KubernetesSpecConfig
-	omsagentImage := "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11012019"
+	omsagentImage := "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod12042019"
 	var workspaceDomain string
 	if cs.Properties.IsAzureStackCloud() {
 		dependenciesLocation := string(cs.Properties.CustomCloudProfile.DependenciesLocation)
 		workspaceDomain = helpers.GetLogAnalyticsWorkspaceDomain(dependenciesLocation)
 		if strings.EqualFold(dependenciesLocation, "china") {
-			omsagentImage = "dockerhub.azk8s.cn/microsoft/oms:ciprod11012019"
+			omsagentImage = "dockerhub.azk8s.cn/microsoft/oms:ciprod12042019"
```

Comment:
just to clarify, the latest image already pushed to dockerhub.cn,right?

Reply:
It is pushed to the public docker hub cloud. I believe there is a sync that happens between public cloud and cn.